src/Appwrite/Specification/Format/Swagger2.php

```diff
@@ -428,7 +428,7 @@ public function parse(): array
                             }
                         }
 
-                        if ($allowed) {
+                        if ($allowed && $validator->getType() === 'string') {
```

Comment:
whats this change supposed to be?

Reply:
This was for SDK Enums, as having non-string enums wasn't supported in most languages. We can removed it but I'm leaning towards keeping it just incase